examples/mnist-rs/README.md

```diff
@@ -0,0 +1,92 @@
+# MNIST-rs
+
+This demo project demonstrates how to train and perform inference in TEE.
+
+## Install the TAs
+
+There are two TAs in the project:
+
+| TA | UUID | Usage |
+| ---- | ---- | ---- |
+| Train | 1b5f5b74-e9cf-4e62-8c3e-7e41da6d76f6 | for training new Model|
+| Inference | ff09aa8a-fbb9-4734-ae8c-d7cd1a3f6744 | for performing reference|
+
+The `Train TA` consumes more memory than `Inference TA`.
+
+Make sure to install them before attempting to use any functions.
+
+## Running the Host
+
+There are three subcommands in the host:
+
+1. Train
+
+    Trains a new model and exports it to the given path.
+
+    ``` shell
+    cargo run -- train -o model.bin
+    ```
+
+    This subcommand downloads the MNIST dataset, trains a new model, and outputs
+    the model to the given path.
+
+    For detailed usage, run: `cargo run -- train --help`.
```

Comment:
If the output isn't too long I'd prefer to include it here so others can see the full usage without needing to build it or checkout the code.